curriculum/challenges/english/25-front-end-development/recipe-page-lab/668f08ea07b99b1f4a91acab.md

```diff
@@ -0,0 +1,198 @@
+---
+id: 668f08ea07b99b1f4a91acab
+title: Build a Recipe Page
+challengeType: 14
+dashedName: build-a-recipe-page
+---
+
+# --description--
+
+**Objective:** Build an app that is functionally similar to <a href="https://mot01.github.io/html-lab-1/" target="_blank" rel="noopener noreferrer nofollow">https://recipe-page.freecodecamp.rocks</a>. Do not copy the demo project, give it your own personal style. Fulfill the user stories below and get all the test to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should have a `!DOCTYPE html` declaration.
+1. You should have an `html` element with `lang` set to `en`.
+1. You should have a `head` element containing a `title` element with the name of your recipe, and a `meta` element with a `charset` attribute set to `UTF-8`.
+1. You should have a `body` element.
+1. You should have an `h1` element with the name of your recipe.
+1. You should have a `p` element that introduces the recipe below the `h1`.
+1. You should have one `h2` element with the text `Ingredients` for the ingredients section.
+1. You should have an unordered list (`ul` element) with at least four list items (`li` elements) that lists your ingredients below the first `h2` element.
+1. You should have a second `h2` element with the text `Instructions` for the instructions section.
+1. You should have an ordered list (`ol` element) with at least four list items that lists the recipe steps in order, below the second `h2`.
+1. You should have one `img` element with a `src` attribute set to a valid image, and an `alt` attribute describing the image. You can use the `cookie-url` if you would like.
```

Comment:
The `cookie-url` be replaced with a cdn url right? Otherwise it is a bit confusing.

Reply:
yes

Reply:
fixed 👍 I have a PR to add a generic image on the CDN repo and used that URL.